tools/Makefile

```diff
@@ -33,9 +33,8 @@ tidy:
 	git diff go.mod go.sum | cat
 	git diff --quiet go.mod go.sum
 
-ci-test-job: failpoint-enable
-	CGO_ENABLED=1 go test ./... -v -tags deadlock -race -cover || { $(MAKE) failpoint-disable && exit 1; }
-	$(MAKE) failpoint-disable
+ci-test-job:
```

Comment:
There is no need to enable failpoint in this command, `ci-test-job` will turn on failpoint at the beginning. 
disabling useless commands here will reduce some of ci time

Reply:
Before we can run `make ci-test-job` in client package independently.

Reply:
Maybe `make test` is better suited for local testing 🤔(although the tools package does not have `make test` now, other Makefiles all have `make test`)  
Or do you think `make test` needs to be merged with `make ci-test-job` as the same command?

Reply:
support `make test` and `make ci-test-job` currently